src/app/views/initial-setup/initial-setup-info/initial-setup-info.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-initial-setup-info',
+  templateUrl: './initial-setup-info.component.html',
+  styleUrls: ['./initial-setup-info.component.css']
+})
+export class InitialSetupInfoComponent implements OnInit {
```

Comment:
wollen wir diesen setupschritt schon mal entfernen?

Reply:
aus routes entfernt, soll die component direkt ganz entfernt werden?

Reply:
jo, dadurch, dass es in der git history weiterhin bleibt, könnten wir es einfach wieder zurückholen (falls nötig)